tracer/src/Datadog.Trace.SourceGenerators/Datadog.Trace.SourceGenerators.csproj

```diff
@@ -285,4 +284,7 @@
       <Link>MessagePack\_last_downloaded_source_url.txt</Link>
     </Content>
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="AspectsDefinitions\Sources\" />
+  </ItemGroup>
```

Comment:
This isn't needed I think

```suggestion
```

Reply:
It is If I want to add the sources to compilation on the fly. Something left for next improvement